### PR TITLE
add quotes to run script phase for spaces

### PR DESCRIPTION
**Pull request by ohayon**

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a changelog summary in the Pull Request description @ohayon</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Review by taquitos — APPROVED** [review]
🎈 🐐

File: deliver/spec/commands_generator_spec.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@ def expect_runner_new_with(expected_options)
   end
 
   describe ":run option handling" do
-    it "can use the username short flag from tool options", now: true do
+    it "can use the username short flag from tool options" true do
```
Comment (KrauseFx) [inline deliver/spec/commands_generator_spec.rb]: The `true` should probably be removed also, right?

Comment (ohayon) [reply deliver/spec/commands_generator_spec.rb]: It was in another commit that @taquitos added to the main PR.

Comment (KrauseFx) [reply deliver/spec/commands_generator_spec.rb]: Thanks for confirming @ohayon 👍

**Pull request merged**